Library/ViewModels/ProjectPamphletMainCellViewModelTests.swift

```diff
@@ -662,13 +631,6 @@ final class ProjectPamphletMainCellViewModelTests: TestCase {
       self.vm.inputs.configureWith(value: (project, .discovery))
       self.vm.inputs.awakeFromNib()
 
-      XCTAssertEqual(self.dataLakeTrackingClient.events, [])
-      XCTAssertEqual(self.segmentTrackingClient.events, [])
-
-      self.vm.inputs.readMoreButtonTapped()
-
-      self.notifyDelegateToGoToCampaignWithProject.assertValues([project])
```

Comment:
Could you explain why you deleted this snippet?

Reply:
Same reason as mentioned above, `self.vm.inputs.readMoreButtonTapped()` would no longer be needed because it's a page viewed event. Same for `notifyDelegateToGoToCampaignWithProject`.